riff-raff.yaml

```diff
@@ -0,0 +1,21 @@
+stacks:
+- discussion
+deployments:
+  static:
+    parameters:
+      cacheControl:
+      - pattern: .*\.js$
+        value: public, max-age=315360000
+      - pattern: .*\.map$
+        value: public, max-age=315360000
+      - pattern: .*
+        value: public, max-age=60
+      mimeTypes:
+        js: application/javascript
+        json: application/json
+        map: text/plain
+      prefixStack: false
```

Comment:
Might be worth making the defaults explicit here that have been removed (e.g. naming prefixes, etc.). Especially the publicACL as that is important for people to be aware of.